pkg/snapshot/publisher.go

```diff
@@ -19,22 +19,27 @@ import (
 	"bytes"
 
 	"github.com/ethereum/go-ethereum/core/types"
-	"github.com/ipfs/go-ipfs-blockstore"
-	"github.com/ipfs/go-ipfs-ds-help"
+	blockstore "github.com/ipfs/go-ipfs-blockstore"
+	dshelp "github.com/ipfs/go-ipfs-ds-help"
+	"github.com/jmoiron/sqlx"
 	"github.com/multiformats/go-multihash"
 
-	"github.com/vulcanize/ipfs-blockchain-watcher/pkg/ipfs/ipld"
-	"github.com/vulcanize/ipfs-blockchain-watcher/pkg/postgres"
```

Comment:
Wow this really dates this repo seeing this old `ipfs-blockchain-watcher` import haha, nice to finally update this!